aws/internal/service/ec2/errors.go

```diff
@@ -20,3 +20,7 @@ const (
 	InvalidVpnGatewayAttachmentNotFound = "InvalidVpnGatewayAttachment.NotFound"
 	InvalidVpnGatewayIDNotFound         = "InvalidVpnGatewayID.NotFound"
 )
+
+const (
+	ErrCodeInternetGatewayIDNotFound = "InvalidInternetGatewayID.NotFound"
```

Comment:
I think the guidance is to match the underlying AWS error code.

```suggestion
	ErrCodeInvalidInternetGatewayIDNotFound = "InvalidInternetGatewayID.NotFound"
```